src/assets/scss/elements/ys-flow-steps.scss

```diff
@@ -4,22 +4,32 @@
 @import '../tools/ys-functions';
 
 .ys-flow-steps {
+  $variant-vertical-ref: &;
+  $bullet-size-px: 34;
   color: $ys-color-grey-28;
 
+  &--vertical {
+    $variant-vertical-ref: &;
+  }
+
   &__list {
     margin: 0;
     padding: 0;
     list-style: none;
     display: flex;
-    flex-flow: row;
     counter-reset: progress-bullet;
     text-align: center;
+
+    #{$variant-vertical-ref} & {

```

Comment:
I think this is difficult to read.

Reply:
It's either that or `.ys-flow-steps--vertical & {`, I guess. I don't have strong opinions about either one - I just tend to use the former. If you prefer the latter I'm happy to remove the refs.